style.css

```diff
@@ -1854,3 +1845,46 @@ form button:focus {
 
   /* contact me section end */
 }
+
+/* mobile toggle section start */
+.mob-icon {
+  display: none !important;
+}
+
+.nav-ul .close-btn {
+  display: none;
+}
+
+.mobile-body {
+  position: fixed;
+  top: 72px;
+  background-color: #3e3c3c;
+  width: 100%;
+  height: 100%;
+  z-index: 99;
+  right: 100%;
+  display: none;
+  transition: all 3s;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
```

Comment:
- [ ] Please do not disable the linters because their purpose is to help write code and maintainable code.👌